diagnostic_rt_qpcr/operation_types/aliquot_positive_template/protocol.rb

```diff
@@ -1,170 +1,162 @@
-  def suspend_lyophilized_RNA(operations)
-    operations.make
-
-    show do 
-      title "Resuspend Positive Template"
-      warning "This reagent should be handled with caution in a dedicated\
+  # Performs the resuspension protocol for a list of operations
+  # that all use the given lyophilized_RNA input.
+  def suspend_lyophilized_RNA()
+    show do
+      title 'Resuspend Positive Template'
+      warning "This reagent should be handled with caution in a dedicated \
       nucleic acid handling area to prevent possible contamination."
-      warning "Freeze-thaw cycles should be avoided. Maintain on ice when\
+      warning "Freeze-thaw cycles should be avoided. Maintain on ice when \
       thawed."
-      check "Resuspend dried Lyophilized Postive Control RNA in each tube\
-      in 1 mL of nuclease-free water to achieve the proper concentration."
+      check "Resuspend dried Lyophilized Postive Control RNA in each tube in \
+      #{qty_display(VOL_WATER)} of nuclease-free water to achieve the \
+      proper concentration."
     end
   end
 
-  #Performs the aliquote protocol for a list of operations
-  #that all use the given lyophilized_RNA input.
-  #
+  # Performs the aliquote protocol for a list of operations
+  # that all use the given lyophilized_RNA input.
   # @param make_aliquots [Item] the lyophilized_RNA
   # @param operations     [OperationList] the list of operations
-  def make_aliquots(operations)
-    last_tube_id = "" #Empty string for storing item id of single use aliquot
-    operations.each do |op|
-      input_RNAs = Array.new(33, op.input("Template").item.id)
-      aliquot_tubes = op.outputs().map{|output| output.item.id}
+  def make_aliquots(ops:, lyophilized_rna:)
+    last_tube_id = '' # Empty string for storing item id of single use aliquot
+    ops.each do |op|
+      input_rnas = Array.new(OUTPUT_ITEMS_NUM[:qty], lyophilized_rna.id)
+      aliquot_tubes = op.outputs.map{ |output| output.item.id }
       transfer_table = Table.new
-              .add_column("Lyophilized RNA", input_RNAs)
-              .add_column("Output RNA Aliquot", aliquot_tubes)
+                            .add_column('Lyophilized RNA', input_rnas)
+                            .add_column('Output RNA Aliquot', aliquot_tubes)
 
       show do
-        title "Aliquot Single Used Aliquot Positive Template"
-        check "Make single use aliquot by transfering 30 L of the diluted\
-        postive control into individual 1.5 mL tubes and label it with the\
-        proper item ID."
-        check "Discard the empty input tube."
+        title 'Aliquot Single Used Aliquot Positive Template'
+        check "Make single use aliquot by transfering \
+        #{qty_display(VOL_SUSPENSION)} of the diluted postive control into \
+        individual #{TUBE_MICROFUGE} and label it with the proper item ID."
+        check 'Discard the empty input tube'
         table transfer_table
```

Comment:
The two-column table here is likely to be confusing because the first column has the same value in every row. The output item number should be sequential, so you could just say something like "Get 33 tubes and label them 12345 - 12377"